src/Transfer/Sources/Appwrite.php

```diff
@@ -934,9 +1050,15 @@ private function exportFunctions(int $batchSize)
     private function exportDeployments(int $batchSize)
     {
         $functionsClient = new Functions($this->client);
-
         $functions = $this->cache->get(Func::getName());
 
+        // exportDeploymentData doesn't exist on Appwrite versions prior to 1.4
+        $appwriteVersion = $this->call('GET', '/health/version', ['X-Appwrite-Key' => '', 'X-Appwrite-Project' => ''])['version'];
```

Comment:
For some reason using a service account returns a 401 since applications don't have the public scope and the version API is on that, so I have to purposely overwrite the key and project to just be a normal non-authenticated request

Reply:
That's interesting 🤔 maybe we should allow public scope for applications as well? Would make sense to me